app/main.py

```diff
@@ -1,2 +1,98 @@
+from __future__ import annotations
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.pairs = [[] for i in range(8)]
+        self.next_index = 0
```

Comment:
Add a separate attribute `length` and return its value in the `__len__` method instead of doing additional calculations as mentioned in the checklist. Change the value of this attribute in operations that manipulate the length.